src/common/meta/src/key.rs

```diff
@@ -78,6 +78,7 @@ use crate::rpc::router::{region_distribution, RegionRoute};
 use crate::DatanodeId;
 
 pub const REMOVED_PREFIX: &str = "__removed";
+pub const MAINTENANCE_KEY: &[u8] = "maintenance_key".as_bytes();
```

Comment:
`_key` suffix of a key is a little redundant.
```suggestion
pub const MAINTENANCE_KEY: &[u8] = "maintenance".as_bytes();
```